### PR TITLE
preliminary celery tasks [#171095912]

### PR DESCRIPTION
**Pull request by uraniumanchor**
# Contributing to the Donation Tracker

- [X] I've added tests or modified existing tests for the change.
- [X] I've humanly end-to-end tested the change by running an instance of the tracker.

### Issue from Pivotal Tracker

https://www.pivotaltracker.com/story/show/171095912

### Description of the Change

This adds two simple Celery tasks, one for testing purposes to ensure Celery is set up correctly (visit `/tracker/celery_test` and you should see the timestamp after ~5 seconds), and one to allow queuing up donation postbacks. Right now the only way to trigger the delayed version is via the admin panel, but eventually the IPN handling code will do this as well.

### Verification Process

Got Celery workers running under Docker Compose, and visited the test page. Got a response and saw the Celery worker log the task completion. Did the same with a test donation and triggered a postback, was able to verify the test endpoint got the payload.

### CODE REVIEW

**Comment by uraniumanchor** [comment]
This is a draft because it relies on #203 being merged first.

File: README.md [hunk 1]
```diff
@@ -24,13 +24,24 @@ For further reading on what else your server needs to look like:
...
+- [Using Celery with Django](https://docs.celeryproject.org/en/stable/django/first-steps-with-django.html) (optional)
+- [Daemonizing Celery](https://docs.celeryproject.org/en/stable/userguide/daemonizing.html) (optional)
 
 Docker should also work but support is still in the experimental phases.
 
 ### Configuration
 
 The Donation Tracker adds a few configuration options.
 
+#### HAS_CELERY
```
Comment (yuna9) [inline README.md:36]: might be nice to put this in the `.env` files in toplevel as a convenience

File: tracker/admin/donation.py [hunk 2]
```diff
@@ -134,6 +135,21 @@ def cleanup_orphaned_donations(self, request, queryset):
...
+        for donation in queryset:
+            if getattr(settings, 'HAS_CELERY', False):
+                from ..tasks import post_donation_to_postbacks
+
+                post_donation_to_postbacks.apply_async(args=(donation.id,))
+            else:
+                from ..eventutil import post_donation_to_postbacks
+
+                post_donation_to_postbacks(donation)
+        self.message_user(request, 'Sent %d postbacks.' % queryset.count())
```
Comment (yuna9) [inline tracker/admin/donation.py:149]: Maybe `Sending` now?

**Review by yuna9 — APPROVED** [review]
epic

File: README.md (hunk 1 above)
Comment (uraniumanchor) [reply README.md:36]: Toplevel is probably not much longer for the world (I don't use it locally any more) otherwise I'd agree with you.

**Pull request merged**